apps/explorer/src/components/placeholder/Table.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import TableCard from '../../components/table/TableCard';
+import placeholdertheme from '../../styles/placeholder.module.css';
+
+type DataType = {
+    rowCount: number;
+    rowHeight: string;
+    colHeadings: string[];
+    colWidths: string[];
```

Comment:
I don't love the approach of using absolute pixel widths here. I wonder if there's a way to use relative percentages and still accomplish something similar?

Reply:
The Figma Design uses absolute pixel widths.